benchmarks/ng-class-bp/app.js

```diff
@@ -0,0 +1,99 @@
+'use strict';
+
+var app = angular.module('ngClassBenchmark', []);
+
+app.controller('DataController', function DataController($scope) {
+
+  this.numberOfTodos = 1000;
+  this.implementation = 'tableOptimized';
+  this.completedPeriodicity = 3;
+  this.importantPeriodicity = 13;
+  this.urgentPeriodicity = 29;
+  this.todos = buildTodos(100);
+  configureTodos(this.todos, 0, this);
```

Comment:
They are helpful.
These lines helped me to make sure that the benchmark does what it means to do, and they are also helpful because now it shows a small preview of the selection that you have done: list, table, single value.
And I do not rebuild the list of 100 todos at the end of the tests to induce a refresh: the list/table is empty and it is restored doing refresh.